src/Tools/GenerateGlobalAnalyzerConfigs/GenerateGlobalAnalyzerConfigs.csproj

```diff
@@ -10,6 +10,7 @@
     <Compile Include="..\..\Microsoft.CodeAnalysis.Analyzers\Core\MetaAnalyzers\ReleaseTrackingHelper.cs" Link="ReleaseTrackingHelper.cs" />
     <Compile Include="..\..\Utilities\Compiler\Debug.cs" Link="Debug.cs" />
     <Compile Include="..\..\Utilities\Compiler\Extensions\PooledHashSetExtensions.cs" Link="PooledHashSetExtensions.cs" />
+    <Compile Include="..\..\Utilities\Compiler\Options\MSBuildItemOptionNames.cs" Link="MSBuildItemOptionNames.cs" />
     <Compile Include="..\..\Utilities\Compiler\Options\MSBuildPropertyOptionNames.cs" Link="MSBuildPropertyOptionNames.cs" />
```

Comment:
JFYI: we are not using/accounting the TFM anymore, not sure if we want to keep it, just letting you know

Reply:
Thanks, I'll leave it for now just in case we need it. Can we removed in a follow-up PR.